client/src/helpers.js

```diff
@@ -49,32 +51,9 @@ const helpers = {
         }
         e.appendChild(o[0]);
     },
-    Z: function() {
```

Comment:
this was used to break malicious proxies that remove the notification modal from the html